api/HTMLAreaElement.json

```diff
@@ -220,10 +220,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": false
```

Comment:
There’s no `HTMLAreaElement.prototype.hreflang` in Blink or Gecko either — and https://html.spec.whatwg.org/multipage/image-maps.html#htmlareaelement doesn’t actually define a `hreflang` member for `HTMLAreaElement`, not even in as an obsolete member in https://html.spec.whatwg.org/multipage/obsolete.html#HTMLAreaElement-partiall. And there is actually no https://developer.mozilla.org/en-US/docs/Web/API/HTMLAreaElement/hreflang article.

So it seems to me we should just remove this entire feature.

Reply:
The data still says Edge and Firefox support this. Not sure if that's right, though.